### DISCUSSION TITLE
wrong output written

### DISCUSSION THREAD

**Discussion by alechash** [denysdovhan/wtfjs#185] [WTF Questions] [answered]
If I am wrong, sorry, I am not the best with JS but trying one of these, I got a different output:
https://github.com/denysdovhan/wtfjs?ref=HackerTabExtension#is-this-multiple-inheritance
I got this: F(0) []
Not this: F []

**Comment by libook** [comment, marked as answer]
I've tested this with Node.js v9,v10,v11,v12,v14.
I've got `F []` with v9,v10,v11.
And I've got `F(0) []` with v12,v14.

So there must be some changes with Node.js 12 (V8 7.4)

**Comment by denysdovhan** [comment]
The questions seem to be answered.
